x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/get_numeric_field_stats.ts

```diff
@@ -158,24 +147,17 @@ export const fetchNumericFieldsStats = (
 
           const stats: NumericFieldStats = {
             fieldName: field.fieldName,
-            count: docCount,
             min: get(fieldStatsResp, 'min', 0),
             max: get(fieldStatsResp, 'max', 0),
             avg: get(fieldStatsResp, 'avg', 0),
             isTopValuesSampled:
               field.cardinality >= SAMPLER_TOP_TERMS_THRESHOLD || samplerShardSize > 0,
```

Comment:
This logic seems wrong now? I don't think you use `SAMPLER_TOP_TERMS_THRESHOLD` anylonger when determining if a thing is sampled or not.